src/App.js

```diff
@@ -1,27 +1,16 @@
 import './App.css';
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+const App = () => (
+  <main className="bg-dark min-vh-100 d-flex flex-column justify-content-center">
+    <div className="container-md">
+      <div className="row mx-0 justify-content-center">
+        <div className="col-md-9">
+          <Calculator />
+        </div>
+      </div>
     </div>
-  );
-}
```

Comment:
- [ ] It seems that you have forgotten to make the App component a class-based component as mentioned in the project requirements below 👇 , kindly let's make it a class-based component 😉 
![image](https://user-images.githubusercontent.com/55465133/137971274-ce32bad1-079b-494f-86a2-d73966398783.png)

Reply:
Hello @barackm, thanks for your review! You're right, I thought it only meant the components I created and not the ones that came by default (the requirements only said "Use class-based components"). I'll gladly make that change and request another review.